### PR TITLE
Add metrics for callstack samples and unwinding errors

### PR DESCRIPTION
**Pull request by florian-kuebler**
Bug: http://b/231552930
Test: Run Test

### CODE REVIEW

File: src/OrbitGl/App.cpp [hunk 1]
```diff
@@ -434,6 +434,15 @@ Future<void> OrbitApp::OnCaptureComplete() {
         ORBIT_CHECK(capture_stopped_callback_);
         capture_stopped_callback_();
 
+        const orbit_client_data::ThreadSampleData* all_threads_sample_data =
+            GetCaptureData().post_processed_sampling_data().GetSummary();
+        if (all_threads_sample_data != nullptr) {
```
Comment (antonrohr) [inline src/OrbitGl/App.cpp:439]: A couple of lines up, `post_processed_sampling_data` is moved into CaptureData. If you do the following lines of code before the move, then this check is unnecessary. 


I am also unsure when the capture metric is sent, so we should definitely test this

**Review by antonrohr — CHANGES_REQUESTED** [review]

File: src/OrbitGl/App.cpp (hunk 1 above)
Comment (florian-kuebler) [reply src/OrbitGl/App.cpp:439]: The check is testing for the result of GetSummary. I actually retrieve the post_processed_sampling_data from capture data, just as in like 430 above. Do you want me to change that?

Comment (antonrohr) [reply src/OrbitGl/App.cpp:439]: I misread this somehow. LGTM and thanks for the explanation

**Comment by florian-kuebler** [comment]
> LGTM as long as it has been tested by you somehow (printf style should be enough)

Added a log in App.cpp:1537 (where we set the capture complete data on a successful capture) and validated that the right numbers are in the metrics_capture_completed_data_.

I'll merge that change as soon as the internal proto is merged.

**Comment by florian-kuebler** [comment]
Merging this now accepting the need of a potential cherry-pick in case this diverges from the internal CL.

**Pull request merged**